projects/archiver/test/invoke/parser.spec.ts

```diff
@@ -27,6 +28,7 @@ describe('parseRecord', () => {
             edition: 'american-edition',
             version: '2019-10-04T16:08:35.951Z',
             issueDate: '2019-10-09',
+            notificationUTCOffset: 1,
         })
     })
 
```

Comment:
`nit` This change will effect the test cases for "'should fail if one of the requiered fields is missing'" (sic). These could possibly be fleshed out to better cover the various scenarious.

Reply:
This is actually covered (including the typo) in a following PR